lib/netutil/conn_pool_test.go

```diff
@@ -59,3 +60,30 @@ func testConnPoolStartStop(t *testing.T, name string, ms *metrics.Set) {
 		}
 	}
 }
+
+func TestGetPutDialConnectionPool(t *testing.T) {
+	t.Skip()
+	ms := metrics.NewSet()
+	dialTimeout := 5 * time.Second
+	compressLevel := 1
+	cp := NewConnPool(ms, "test-pool", "127.0.0.1:8401", handshake.VMSelectClient, compressLevel, dialTimeout, 0)
+
+	connChan := make(chan *handshake.BufferedConn, 50)
+	wg := sync.WaitGroup{}
+	for i := 0; i < 50; i++ {
+		wg.Add(1)
+		go func() {
+			conn, err := cp.Get()
+			if err != nil {
+				t.Errorf("expecting non-nil error after MustStop(), err:%v", err)
```

Comment:
The message does not make sense to me. Is it a typo?

Reply:
Yes it's fixed now.